frontend/tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
     "compilerOptions": {
-        "target": "ES2020",
+        "target": "ESNext",
```

Comment:
what is the cause of this change

Reply:
I might have an idea what caused this, but that's the abstracted header file that has been removed